bash/bashrc

```diff
@@ -5,6 +5,21 @@
 # If not running interactively, don't do anything
 [[ $- != *i* ]] && return
 
+## Imma Let You Finish But
+## =======================
+
+osascript << LOL
+set trololol to "https://www.youtube.com/watch?reload=9&v=dQw4w9WgXcQ"
```

Comment:
Prefer `https://www.youtube.com/watch?v=L_jWHffIx5E` for all trolling going forward.

Reply:
Good point, this is far more idiomatic